File: 3740-minimum-distance-between-three-equal-elements-i/3740-minimum-distance-between-three-equal-elements-i.py
class Solution:
    def minimumDistance(self, nums: List[int]) -> int:
        count=100000
        for i in range(len(nums)-2):
            for j in range(i+1, len(nums)-1):
                if nums[i]==nums[j]:
                    for k in range(j+1, len(nums)):
                        if nums[j]==nums[k]:
                            val=(j-i)+(k-i)+(k-j)
                            if val<count:
                                count=val
                                break

        if count==100000:
            return -1
        return count

File: 3740-minimum-distance-between-three-equal-elements-i/3740-minimum-distance-between-three-equal-elements-i.java
class Solution {
    public int minimumDistance(int[] nums) {
        int count=Integer.MAX_VALUE;
        for(int i=0; i<nums.length-2; i++){
            for(int j=i+1; j<nums.length-1; j++){
                if(nums[i]==nums[j]){
                    for(int k=j+1; k<nums.length; k++){
                        if(nums[j]==nums[k]){
                            int val=(j-i)+(k-j)+(k-i);
                            if(val<count)count=val;
                            break;
                        }
                    }
                }
            }
        }
        if(count==Integer.MAX_VALUE)return -1;
        return count;
    }
}